source/common/buffer/buffer_impl.cc

```diff
@@ -390,9 +387,9 @@ ReservationSingleSlice OwnedImpl::reserveSingleSlice(uint64_t length, bool separ
   if (reservable_size >= length) {
     reservation_slice = slices_.back().reserve(length);
   } else {
-    Slice slice(length, account_);
```

Comment:
just a question, the Slice can account the memory, the SizedStorage won't. Is that something we needn't for reservation?

Reply:
Yes, we will account the memory usage when we commit this reservation.